src/components/Button.jsx

```diff
@@ -1,5 +1,19 @@
+import React from "react";
+import "tailwindcss/tailwind.css";
+
 const Button = () => {
-  return <div>Button</div>;
+  return (
+    <button
+      className="bg-rishi-white border border-rishi-orange text-rishi-black font-urbanist py-2 px-4 rounded"
```

Comment:
can you add border-2 px-6 rounded-md and get rid of py-2?

also make it so when you hover your mouse over the button, the bg turns bg-rishi-orange
https://tailwindcss.com/docs/hover-focus-and-other-states

Reply:
![Uploading Screenshot 2024-01-13 at 6.29.49 PM.png…]()